example/components/ProxyExample/ProxiedUserInfoForm.tsx

```diff
@@ -0,0 +1,63 @@
+import React, { useState } from 'react';
+import { ProxyContext, useStockContext } from 'stocked';
+import { proxyContext } from './ProxyExample';
+
+const UserInfoForm = () => {
+    // Get proxied context (ProxyContext)
+    const { getValue, setValue } = useStockContext();
+
+    // Proxied paths to values
+    const proxiedNamePath = 'userInfo.name';
+    const proxiedSurnamePath = 'userInfo.surname';
+    const proxiedRegNoPath = 'userInfo.regData.no';
+
+    // States for values in form
+    const [name, setName] = useState<string>(getValue<string>(proxiedNamePath));
+    const [surname, setSurname] = useState<string>(getValue<string>(proxiedSurnamePath));
+    const [regNo, setRegNo] = useState<number>(getValue<number>(proxiedRegNoPath));
```

Comment:
Don't use proxy like that - `useStockState` works better, it automatically synchronizes value with all values. Also, it applies proxy.